src/com/github/johnthagen/cppcheck/CppcheckInspection.java

```diff
@@ -58,17 +61,14 @@
                     CppCheckInspectionImpl.executeCommandOnFile(cppcheckPath, prependIncludeDir(cppcheckOptions, vFile),
                             tempFile.getAbsolutePath(), cppcheckMisraPath);
 
-            if (!cppcheckOutput.isEmpty()) {
-                final List<ProblemDescriptor> descriptors = CppCheckInspectionImpl.parseOutput(file, manager, document, cppcheckOutput,
-                        tempFile.getName());
-                return descriptors.toArray(new ProblemDescriptor[0]);
-            }
-        } catch (ExecutionException | CppcheckError | IOException ex) {
+            final List<ProblemDescriptor> descriptors = CppCheckInspectionImpl.parseOutput(file, manager, document, cppcheckOutput,
```

Comment:
I removed the check since this should never been empty and if it is we want to get an error about it.